geopolars/src/geoseries.rs

```diff
@@ -169,6 +172,23 @@ impl GeoSeries for Series {
         Series::try_from(("geometry", Arc::new(result) as ArrayRef))
     }
 
+    fn from_geom_vec(geoms: &Vec<Geometry<f64>>) -> Result<Self> {
+        let mut wkb_array = MutableBinaryArray::<i32>::with_capacity(geoms.len());
+
+        for geom in geoms {
+            let wkb = geom.to_wkb(CoordDimensions::xy()).map_err(|_| {
+                PolarsError::ComputeError(std::borrow::Cow::Borrowed(
+                    "Failed to convert geom vec to GeoSeries",
+                ))
+            })?;
+            wkb_array.push(Some(wkb));
+        }
+        let array: BinaryArray<i32> = wkb_array.into();
+
+        let series = Series::try_from(("geometry", Arc::new(array) as ArrayRef)).unwrap();
+        Ok(series)
+    }
+
```

Comment:
@kylebarron added this convenience function to more easily build GeoSeries. Let me know if it looks good and we can probably use it a few other places in the codebase to clean up the code a little.

Tried to make it a TryFrom implementation but cant implement the Trait for Series as it's coming from the Polars crate